lib/ocpp/v201/charge_point.cpp

```diff
@@ -74,7 +75,14 @@ ChargePoint::ChargePoint(const std::map<int32_t, int32_t>& evse_connector_struct
         EVLOG_AND_THROW(std::invalid_argument("Database handler should not be null"));
     }
 
+    if (!this->message_queue) {
```

Comment:
Not sure if this will work. The constructor below calls this with a nullptr so this will always throw. Might need a different solution to this issue.